terraform/aws-linux/main.tf

```diff
@@ -0,0 +1,137 @@
+variable "num_nodes" {
+	default = 1
+}
+
+variable "hostname_formatter" {
+	default = "amz-%d"
+}
+
+variable "ssh_user" {
+	default = "gila"
```

Comment:
Should this be `var.ssh_user` to pick up the one defined in the top-level variables.tf?

Reply:
No, it does not use the module system at all. Its "stand-alone"